fix(l10n): translations update from weblate.recipesage.com

recipesage-weblate opened the pull request:
Translations update from [RecipeSage Weblate](http://weblate.recipesage.com) for [RecipeSage App/Frontend UI](http://weblate.recipesage.com/projects/recipesage-app/app-frontend/).



Current translation status:

![Weblate translation status](http://weblate.recipesage.com/widget/recipesage-app/app-frontend/horizontal-auto.svg)

Commit:
fix(l10n): translated using Weblate (Danish)

Currently translated at 100.0% (982 of 982 strings)

Translation: RecipeSage App/Frontend UI
Translate-URL: http://weblate.recipesage.com/projects/recipesage-app/app-frontend/da/

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/julianpoy/RecipeSage?pullRequest=1304) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/julianpoy/RecipeSage?pullRequest=1304) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/julianpoy/RecipeSage?pullRequest=1304) it.</sub>

recipesage-weblate committed:
fix(l10n): update translation files

Updated by "Cleanup translation files" hook in Weblate.

Translation: RecipeSage App/Frontend UI
Translate-URL: http://weblate.recipesage.com/projects/recipesage-app/app-frontend/
